src/main/java/se/skvf/kaninregister/api/BunnyRegistryApiImpl.java

```diff
@@ -407,14 +407,33 @@ private BunnyList toBunnyList(Collection<Bunny> bunnies) {
 		list.setBunnies(bunnies.stream().map(BunnyRegistryApiImpl::toListDTO).collect(Collectors.toList()));
 		return list;
 	}
-	
+
+	@Override
+	public OwnerDTO session() {
+		return process(() -> {
+			String session = getSession();
+			if (session == null) {
+				return null;
+			}
+
+			String ownerId = sessions.getOwnerIdForSession(session);
+			if (isBlank(ownerId)) {
+				return null;
+			}
+
+			return toDTO(validateOwner(ownerId, false));
+		});
+	};
+
 	@Override
 	public OwnerDTO login(LoginDTO loginDTO) {
 		return process(() -> {
 
-			if (getSession() != null) {
-				throw new WebApplicationException(CONFLICT);
+			String session = getSession();

```

Comment:
Ska vi hantera detta på något annat sätt? Det löser ett edge-case. En client har kakor kvar i browsern men sessionen är borta i servern, t.ex. efter en omstart av servern. Om man t.ex. refreshar browsern blir man redirektad till /login men när man försöker logga in får man "du är redan inloggad" och kommer inte vidare.

Reply:
Men om kakan finns så hamnar du väl på /bunnies och inte /login? Och sedan får du 401 **ej** inloggad som browsern får ta. Då bör ju dock servern ta bort kakorna varje gång den inte hittar någon session.
Jag skapar ett ärende.